services/service.js

```diff
@@ -8,30 +8,27 @@ mongoose.connect('mongodb://test:test@ds257627.mlab.com:57627/first-blog-mern',
 
 class service {
 
+    constructor()
+    {
+        this.Data = {};
+    }
+
+    
+    getOnePost(req) {
```

Comment:
getOnePost should not take in `req`, but rather it should take in `post_id`

Reply:
fixed